Scripts/Install-iLaps_v2.0.ps1

```diff
@@ -158,7 +159,8 @@ foreach ($installer in $installers) {
     try {
         # Remove existing scripts.
         Remove-Item ($Path + "\" + $Installer) -ErrorAction Ignore;
-        
+        # Remove V1 Scripts if they exist
+        Remove-Item ($V1CleanupPath + "\" + $Installer) -ErrorAction Ignore;
```

Comment:
@TheronHowton mentioned this would fail due to new name of file's for V2, will need fixing